### PR TITLE
Feature/alcs 526

### PR DESCRIPTION
**Pull request by mhuseinov**
custom stepper for application review

### CODE REVIEW

File: portal-frontend/src/app/features/review-application/review-application.component.html [hunk 1]
```diff
@@ -5,19 +5,13 @@ <h6 *ngIf="application">Application ID: {{ application.fileNumber }} | {{ applic
...
 </ng-template>
 
-<mat-stepper *ngIf="!isFirstNationGovernment" class="review-application" linear="true" #stepper>
+<!-- 
+
+<mat-stepper *ngIf="isFirstNationGovernment" class="review-application" linear="true" #stepper>
   <mat-step label="Contact Information">
     <ng-container *ngTemplateOutlet="header"></ng-container>
     <app-review-contact-information></app-review-contact-information>
   </mat-step>
```
Comment (dhaselhan) [inline portal-frontend/src/app/features/review-application/review-application.component.html:14]: Can we remove the commented out code?

File: portal-frontend/src/app/features/review-application/review-application.component.spec.ts [hunk 2]
```diff
@@ -50,6 +51,7 @@ describe('ReviewApplicationComponent', () => {
         },
       ],
       declarations: [ReviewApplicationComponent],
+      imports: [MatSnackBarModule],
```
Comment (dhaselhan) [inline portal-frontend/src/app/features/review-application/review-application.component.spec.ts:54]: Should create and provide a DeepMocked<ToastService. instead.

File: portal-frontend/src/app/features/review-application/review-application.component.html (hunk 1 above)
Comment (mhuseinov) [reply portal-frontend/src/app/features/review-application/review-application.component.html]: sure, thanks!

**Review by dhaselhan — APPROVED** [review]
LGTM 👍

**Pull request merged**